server/test/unit/vitalsTest/checkForInternalProblemsTest.js

```diff
@@ -0,0 +1,215 @@
+/* eslint-disable no-plusplus */
```

Comment:
Reminder that we spoke about moving this file into the `integration` folder :smile: